AudioKit/Common/Nodes/Playback/Samplers/Sampler/AKSamplerDSP.mm

```diff
@@ -277,6 +277,14 @@
     }
 }
 
+void AKSamplerDSP::processWithEvents(const AudioTimeStamp *timestamp, AUAudioFrameCount frameCount, const AURenderEvent *events)
+{
+    for (int i = 0; i < _outBufferListPtr->mNumberBuffers; i++) {
+        memset(_outBufferListPtr->mBuffers[i].mData, 0, _outBufferListPtr->mBuffers[i].mDataByteSize);
```

Comment:
@getdunne Here's where I'm zeroing things out.

Reply:
Another option would be to change the sampler’s internals so that it does not generate additively.

Reply:
@dave234 I thought about that, but since BufferedAudioUnit is already all about buffer management, it makes sense to zero the output buffer there for two reasons:
1. Additive generation is the most natural approach for all multi-voice instrument AUs.
2. It ensures that a "null" AU (which does no DSP at all) will output silence.

Reply:
@dave234 I will test myself this morning and approve this PR if no problems. BTW thanks for tracking this down.

Reply:
@dave234 I just looked again and realized you're simply adding buffer zeroing to my code. I can do that at a lower level in AKCoreSampler.cpp, with no need to add a processWithEvents() member to AKSamplerDSP.